src/zrc/api/viewsets.py

```diff
@@ -322,9 +333,11 @@ class ZaakObjectViewSet(NotificationCreateMixin,
         'create': SCOPE_ZAKEN_CREATE,
     }
     notifications_kanaal = KANAAL_ZAKEN
+    audit = AUDIT_ZRC
 
 
 class ZaakInformatieObjectViewSet(NotificationCreateMixin,
+                                  AuditTrailViewsetMixin,
```

Comment:
this is adding the `update` to `ZaakInformatieObject`, which isn't needed/allowed here (no `partial_update` or `update` actions).

```suggestion
                                  AuditTrailCreateMixin,
                                  AuditTrailDestroyMixin,
```